cip/CIP2015-10-12-CREATE.adoc

```diff
@@ -0,0 +1,136 @@
+= CIP2015-10-12 - CREATE
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Authors:* Pontus Melke pontus.melke@neotechnology.com, Mats Rydberg mats@neotechnology.com
+
+[abstract]
+.Abstract
+--
+This proposal formalizes `CREATE` which is used in Cypher to create nodes and relationships.
+--
+
+toc::[]
+
+== Motivation & Background
+`CREATE` is how nodes and relationships are created in Cypher.
+The syntax has been around since Cypher 1.7 but never properly formalized.
+
+== Proposal
+
+`CREATE` is used to create new nodes and relationships into the graph, as well as set initial labels, type, and properties on the new entities.
```

Comment:
"create new nodes and relationships into the graph" -> "add new nodes and relationships to the graph"